lint/scripts/beta-check.sh

```diff
@@ -0,0 +1,6 @@
+set -e
+
+if egrep "@turo\/.*\-beta\-" package.json; then
```

Comment:
I wouldn't hardcode `@turo` here, but instead make this an input of the action instead

Reply:
good catch